index.js

```diff
@@ -19,5 +19,24 @@ module.exports = function(source) {
         this.cacheable();
     }
 
+    // if force sourcemap requested embed inline sources (helps resolve issues with
+    // transpiled languages like typescript
+    if (userOptions['force-sourcemap']) {
```

Comment:
From my understanding, the second parameter (`sourceMap`) will be `null` or `undefined` unless a previous loader in the chain has provided one. Most transpiling loaders provide the `sourceMap` parameter, but for third-party code (ex. Angular 2, bootstrap, etc.) you have to use `source-map-loader`. `source-map-loader` will load both inline and external source maps, which should solve the problem you were seeing. Even so, if the second parameter of the loader isn't provided, a `sourceMappingURL` shouldn't be generated for the code since it will only cause webpack to process an object that it shouldn't even have to look at.